src/bp-forums/common/template.php

```diff
@@ -1617,7 +1617,7 @@ function bbp_get_dropdown( $args = '' ) {
 	// Items found so walk the tree
 	if ( ! empty( $posts ) ) {
 		add_filter( 'list_pages', 'bbp_reply_attributes_meta_box_discussion_reply_title', 999, 2 );
-		unset( $r['walker'] );
+		//unset( $r['walker'] );
```

Comment:
```suggestion
		// unset( $r['walker'] );
```

Reply:
@chetansatasiya 
How do i disabled specific option from forum drop-down without the above 'walker' parameter?